apps/api/.env.example

```diff
@@ -19,6 +19,9 @@ MAX_RETRY_COUNT=3 # Max retry count. DEFAULT: 3
 ARCHIVE_LIMIT=1000 # Max notifications to archive. DEFAULT: 1000
 ARCHIVE_INTERVAL=3600 # Interval (in seconds) for archiving notifications. DEFAULT: 3600 (every 1 hour)
 SCHEDULE_TIME_IN_SECONDS=5 # Interval (in seconds) for processing PENDING and AWAITING_CONFIRMATION notifications. DEFAULT: 5 (every 5 seconds)
+ENABLE_ARCHIVED_NOTIFICATION_DELETION=false # Toggle to enable/disable deletion of archived notifications, VALUES: "true", "false". DEFAULT: false
+DELETE_INTERVAL=2592000 # Interval (in seconds) for deletion of archived notifications. DEFAULT: 2592000 (every 30 days)
```

Comment:
This is like when to trigger this function right? Can we not just use the cron pattern here instead of something like this?